packaging/examples/metrics/grafana-dashboards/strimzi-kafka.json

```diff
@@ -1015,7 +1015,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(kubelet_volume_stats_available_bytes{namespace=\"$kubernetes_namespace\",persistentvolumeclaim=~\"data(-[0-9]+)?-$strimzi_cluster_name-kafka-[0-9]+\"}) by (persistentvolumeclaim)",
+          "expr": "sum(kubelet_volume_stats_available_bytes{namespace=\"$kubernetes_namespace\",persistentvolumeclaim=~\"data(-[0-9]+)?-$strimzi_cluster_name-(kafka|$pool_name)-[0-9]+\"}) by (persistentvolumeclaim)",
```

Comment:
it is set in templating, lines 2976 - 2996

Reply:
But based on what?

Reply:
There is a query `query_result(kafka_server_replicamanager_leadercount{namespace=\"$kubernetes_namespace\"})` which returns some metrics, from this result we filter `"/.*strimzi_io_pool_name=\"([^\"]*).*/"`, in case it is not part of the metrics, the pool_name will be simply empty and in that case `kafka` should be used for matching the PVC name.

Reply:
I removed it from the top menu as it doesn't make sense to me to have it there because it influences only one metric. So the result of it will always be a list of available nodepool collected via strimzi_io_pool_name

Reply:
Ok, thanks for the explanation. Maybe its worth explaining in the description?

Reply:
ye, I will add it :)